Embedded Metadata.js

```diff
@@ -795,15 +793,28 @@ function finalDataCleanup(doc, newItem) {
 		// because most of the time they are not right
 		newItem.tags = [];
 	}
-	
+
 	//Cleanup DOI
 	if (newItem.DOI){
 		newItem.DOI =newItem.DOI.replace(/^doi:\s*/, "");
 	}
-	
+
+	// Add DOI to non-supported item types
+	if (newItem.DOI && !ZU.fieldIsValidForType("DOI", newItem.itemType)) {
+		if (newItem.extra){
+			newItem.extra += "\nDOI: " + newItem.DOI;
```

Comment:
Since there is no way for the Extra field to contain anything other than note (given the CSL import from Zotero) I just empty the field above and can do without all this code.